### DISCUSSION TITLE
Issues in Arabic playlist

### DISCUSSION THREAD

**Discussion by iptvchecker** [iptv-org/community#121] [Show and tell]
### Stream Title

22 Arab countries channels streams copy/update to ara.m3u!

### What happened to the stream?

Channels belonging to the Arabian diaspora do not appear in the correct sections
Certain religious channels do not seem to work.

### Notes (from author)

https://iptv-org.github.io/iptv/languages/ara.m3u is gutted out! Many of the channels that used to be there do not exist anymore! For example, I reported that "Rotana Classic" was removed! I provided the proper link from a previous ara.m3u list that had it.

The channel was added to the SA.m3u. It's true that Rotana channels belong to the Saudi Arabia list, but also KSA is part of the Arabic speaking list! Can you, pretty please, add that channel to the ara.m3u list?

Also, channels, like Aljazeera Arabic is missing from ara.m3u, but in Qatar list! No idea why this ara.m3u was emptied so bad like this!

If I may suggest that someone goes through the 22 Arab countries' lists and do an auto-copy of their channel streams into the Arabic speaking countries list ara.m3u.

Also, ALMOST all the Islamic channels are NOT working. We're coming to the holly month of Ramadan and it would be great if you can put these channels back.

Obviously all my suggestions are polite requests  and no assumption or pressure is assumed!

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by BellezaEmporium** [comment]
- Some channels had an workaround to make them available and belonged to an old active user of this repository. I can change them, i'll work on it and give a patch later.
- Please verify if those channels are in the MENA regional playlist, if they're not, there's an issue.

**Comment by iptvchecker** [comment]
There is another arab.m3u in regions (https://iptv-org.github.io/iptv/regions/arab.m3u) It does not have channels like Makkah TV!

It seems that there are multiple channels for Arabic, but they're not synced. I suggest all the Arabic channels in their countries be updated and synced. I noticed that Islamic (Sunni) Channels are disappearing. Either removed or not working. There are Christians and Shiite, but their a minority (with all due respect!) There are many working channels for these minority groups and Main Stream Islam has fewer and fewer. Can someone who's Sunni Muslim lend a hand to provide better content?

Al-Ressalah is not working, Makkah TV does not exist, both Live Quran and  Sunnah (Makkah and Maddina) channels are not working!

Thank you and I appreciate your help with this content.

**Comment by BellezaEmporium** [comment]
> There is another arab.m3u in regions (https://iptv-org.github.io/iptv/regions/arab.m3u) It does not have channels like Makkah TV!
> 
> It seems that there are multiple channels for Arabic, but they're not synced. I suggest all the Arabic channels in their countries be updated and synced. I noticed that Islamic (Sunni) Channels are disappearing. Either removed or not working. There are Christians and Shiite, but their a minority (with all due respect!) There are many working channels for these minority groups and Main Stream Islam has fewer and fewer. Can someone who's Sunni Muslim lend a hand to provide better content?
> 
> Al-Ressalah is not working, Makkah TV does not exist, both Live Quran and  Sunnah (Makkah and Maddina) channels are not working!
> 
> Thank you and I appreciate your help with this content.

I have found Saudi Sunnah & Saudi Quran TV while looking after an Indonesian host. As for the others, I cannot tell as it's another kettle with the fish.

**Comment by iptvchecker** [comment]
here is a link for Makkah TV. Not sure if this is something you can embed! https://makkahtv.tv/

**Comment by BellezaEmporium** [comment]
> here is a link for Makkah TV. Not sure if this is something you can embed! https://makkahtv.tv/

https://media2.streambrothers.com:1936/8122/8122/playlist.m3u8

**Comment by iptvchecker** [comment]
Excellent! I tried the link you provided for Makkah TV. It works great!

I appreciate that you're taking care of this list. Of course, if you can find a similar stream for Al-Ressalah. https://rotana.net/?tv=live-elressalah. The only available one is a geo-blocked one.

Many thanks!

**Comment by iptvchecker** [comment]
I went through the different lists.

It seems that there are Arab world list, MENA list, Morocco list. Maybe there are more. This is in the Region list. Also, in the languages there is the Arabic ara list. This was the first one I was talking to you guys about.

It seems that there are multiple lists to provide streams for Arabic countries but they're not in sync. 

I'd suggest that in the region list you concentrate the streams in one, for the Arab channels. If you say MENA then it's middle east (including Israel which means Hebrew content) and Europe channels, which is a huge set! Maybe Arabic region would include ONLY the 22 countries that are "part of the Arab League" if we're identify countries as Arabic.

It's confusing that there are too many lists and none is complete, synced or working! 

It's Ramadan time and the channels that address Islamic content would be great to have!

Thank you again and I appreciate your help.

Hopefully my comments are helpful in maintaining these Arabic and Islamic lists!

**Comment by iptvchecker** [comment]
How can these issues be tracked if the ticket is converted to a discussion?
